support-frontend/app/controllers/SubscriptionProductCookiesCreator.scala

```diff
@@ -32,47 +32,25 @@ case class SubscriptionProductCookiesCreator(domain: GuardianDomain) {
     )
   }
 
-  def createCookiesForProduct(product: ProductType): List[Cookie] = {
+  def createCookiesForProduct(product: ProductType, now: DateTime): List[Cookie] = {
     // Setting the user attributes cookies used by frontend. See:
     // https://github.com/guardian/dotcom-rendering/blob/3c4700cae532993ace6f40c3b59c337f3efe2247/dotcom-rendering/src/client/userFeatures/user-features.ts
     val standardCookies: List[SupportCookie] = List(
-      SessionCookie("gu_user_features_expiry", DateTime.now.plusDays(1).getMillis.toString),
+      SessionCookie("gu_user_features_expiry", now.plusDays(1).getMillis.toString),
       SessionCookie("gu_hide_support_messaging", true.toString),
     )
+
     val oneWeekInSeconds = 604800
+
     val productCookies: List[SupportCookie] = product match {
-      case Contribution(_, _, billingPeriod) =>
-        List(
-          SessionCookie(
-            s"gu.contributions.recurring.contrib-timestamp.$billingPeriod",
-            DateTime.now.getMillis.toString,
-          ),
-          SessionCookie("gu_recurring_contributor", true.toString),
-        )
-      case _: SupporterPlus =>
-        List(
-          SessionCookie("gu_digital_subscriber", true.toString),
-          // "gu_supporter_plus" -> true.toString, // TODO: add this and remove the digisub one now that the CMP cookie list has been updated
-          SessionCookie("GU_AF1", DateTime.now().plusDays(1).getMillis.toString),
-        )
-      case _: TierThree =>
-        List(
-          SessionCookie("gu_digital_subscriber", true.toString),
-          // SessionCookie("gu_supporter_plus", true.toString), // TODO: add this and remove the digisub one now that the CMP cookie list has been updated
-          SessionCookie("GU_AF1", DateTime.now().plusDays(1).getMillis.toString),
-        )
-      case _: DigitalPack =>
-        List(
-          SessionCookie("gu_digital_subscriber", true.toString),
-          SessionCookie("GU_AF1", DateTime.now().plusDays(1).getMillis.toString),
-        )
+      case _: SupporterPlus | _: TierThree | _: DigitalPack =>
+        List(SessionCookie("GU_AF1", now.plusDays(1).getMillis.toString))
+
       case p: Paper if p.productOptions.hasDigitalSubscription =>
```

Comment:
I know this is pre-existing but we give ad-free to all paper subscribers now, not just those with the digipack add on